CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [0.4.3]
++ Error out of RAiDER when fetching weather model if request to API fails; previously would skip datetime and continue
```

Comment:
I think this should be reverted. Only error out in the workflow that is called by HYP3.